docs/deploy.zh-CN.md

```diff
@@ -102,6 +102,14 @@ $ ./api/run.sh &
 $ kill $(ps aux | grep 'manager-api' | awk '{print $2}')
 ```
 
+## 打包
+
+你可以把刚刚 output 目录整体打包，copy 到其他地方解压运行，output 目录包含运行 dashboard 需要的所有文件(配置文件、可执行文件、web静态资源)
+
+```sh
+$ tar –cvf dashboard.tar ./output/*
```

Comment:
should use `makefile`, take a look at:

https://github.com/apache/apisix/blob/master/Makefile#L200

Reply:
ok